optax/_src/transform_test.py

```diff
@@ -202,6 +202,35 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  @parameterized.parameters(
+    (0.01, 0.55, 123),
+    (0.3, 0.54, 234),
+    (1.0, 0.56, 314),
+  )
+  def test_add_noise_has_correct_variance_scaling(self, eta, gamma, seed):
```

Comment:
Added the variants.

Agreed there is nothing new the other tests can catch. Kwargs there don't work because the parameters are reused elsewhere (seed for the other transform, eta and gamma for scale), so I've just hard coded them as variables.